listenbrainz/webserver/templates/footer.html

```diff
@@ -0,0 +1,323 @@
+<section class="footer">
+    <div class="container-fluid">
+        <div class="row mb-4 mt-2 pt-2">
+            <div class="col-md-6">
+                <h3 class="ms-3">
+                    <img src="{{ url_for('static', filename='img/listenbrainz-logo.svg') }}" width="196" alt="ListenBrainz"/>
+                </h3>
+                <br />
+                <p class="ms-3">
+                    You can use ListenBrainz to track your listening habits, discover new music with personalized recommendations, and share your musical taste with others using our visualizations.
+                </p>
+                <ul class="list-unstyled ms-3">
+                    <li>
+                <span>
+                  Development IRC:
+                </span>
+                        <a
+                                class="fw-bold ms-1"
+                                href="https://kiwiirc.com/nextclient/irc.libera.chat/?#metabrainz"
+                                rel="noopener noreferrer"
+                                target="_blank"
+                        >
+                            #metabrainz
+                        </a>
+                    </li>
+                <span>
+                  Forums:
+                </span>
+                        <a
+                                class="fw-bold  ms-1"
+                                href="https://community.metabrainz.org"
+                                rel="noreferrer"
+                                target="_blank"
+                        >
+                            community.metabrainz.org
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="col-sm-12 col-md-3">
+                <h3 class="ms-3">
+                    Fellow Projects
+                </h3>
+                <ul class="list-style-type">
```

Comment:
Not sure what styles would need to be transferred from before, but I quite liked the chevrons you used in the footer:
<img width="188" alt="image" src="https://user-images.githubusercontent.com/6179856/171157475-e9182be5-5fd6-4d8c-8991-9fff8b0d47d9.png">

Reply:
Bitmap left a reply suggesting to me that we shouldn't add extra styles and hence I should remove the chevrons and keep things standard. I do think that makes sense, hence made the move. Let me know if that if fine.

Reply:
Or perhaps removing the list style entirely?

<img width="643" alt="image" src="https://user-images.githubusercontent.com/6179856/171440216-635b39df-4817-4104-80d4-ff3acbfee978.png">